packages/@vuepress/plugin-last-updated/index.js

```diff
@@ -10,6 +10,7 @@ module.exports = (options = {}, context) => ({
         ? transformer(timestamp, $lang)
         : defaultTransformer(timestamp, $lang)
       $page.lastUpdated = lastUpdated
+      $page.lastUpdatedTstamp = timestamp
```

Comment:
I think we can add the full name here lastUpdatedTimestamp lastUpdatedTimeStamp for me it could a lot more clearer

Reply:
Imho is tstamp a very common term for the Unix (like) time stamp but I will adjust it

Reply:
Maybe others can answer too. Actually i dont see any short write of variable inside vuepress.